modules/oadp-installing-dpa-1-3.adoc

```diff
@@ -0,0 +1,384 @@
+<13> Specify a prefix for Velero backups, for example, `velero`, if the bucket is used for multiple purposes.
+endif::[]
+ifdef::virt-installing-configuring-oadp[]
++
+[source,yaml,subs="attributes+"]
+----
+apiVersion: oadp.openshift.io/v1alpha1
+kind: DataProtectionApplication
+metadata:
+  name: <dpa_sample>
+  namespace: openshift-adp
+spec:
+  configuration:
+    velero:
+      defaultPlugins:
+        - kubevirt <1>
+        - gcp <2>
+        - csi <3>
+        - openshift <4>
+      resourceTimeout: 10m <5>
+    nodeAgent: <6>
+      enable: true <7>
+      uploaderType: kopia <8>
+      podConfig:
+        nodeSelector: <node_selector> <9>
+  backupLocations:
+    - velero:
+        provider: {provider} <8>
+        default: true
+        credential:
+          key: cloud
+          name: <default_secret> <9>
+        objectStorage:
+          bucket: <bucket_name> <10>
+          prefix: <prefix> <11>
+----
+<1> The `kubevirt` plugin is mandatory for {VirtProductName}.
+<2> Specify the plugin for the backup provider, for example, `gcp`, if it exists.
+<3> The `csi` plugin is mandatory for backing up PVs with CSI snapshots. The `csi` plugin uses the link:https://{velero-domain}/docs/main/csi/[Velero CSI beta snapshot APIs]. You do not need to configure a snapshot location.
+<4> The `openshift` plugin is mandatory.
+<5> Specify how many minutes to wait for several Velero resources before timeout occurs, such as Velero CRD availability, volumeSnapshot deletion, and backup repository availability. The default is 10m.
+<6> The administrative agent that routes the administrative requests to servers.
+<7> Set this value to `true` if you want to enable `nodeAgent` and perform File System Backup.
+<8> Enter `kopia` or `restic` as your uploader. You cannot change the selection after the installation. The `nodeAagent` deploys a daemon set, which means that the `nodeagent` pods run on each working node. You can configure File System Backup by adding `spec.defaultVolumesToFsBackup: true` to the `Backup` CR.
+<9> Specify the nodes on which Kopia or Restic are available. By default, Kopia or Restic run on all nodes.
+<10> Specify the backup provider.
+<11> Specify the correct default name for the `Secret`, for example, `cloud-credentials-gcp`, if you use a default plugin for the backup provider. If specifying a custom name, then the custom name is used for the backup location. If you do not specify a `Secret` name, the default name is used.
+<12> Specify a bucket as the backup storage location. If the bucket is not a dedicated bucket for Velero backups, you must specify a prefix.
+<13> Specify a prefix for Velero backups, for example, `velero`, if the bucket is used for multiple purposes.
+endif::[]
+
+. Click *Create*.
+. Verify the installation by viewing the OADP resources:
++
+[source,terminal]
+----
+$ oc get all -n openshift-adp
+----
++
+.Example output
++
```

Comment:
`NAME                                                    READY   STATUS    RESTARTS   AGE
pod/node-agent-729nn                                    1/1     Running   0          26s
pod/node-agent-wspmz                                    1/1     Running   0          26s
pod/node-agent-xg9pr                                    1/1     Running   0          26s
pod/openshift-adp-controller-manager-58558fcf5f-l2v8x   1/1     Running   0          9h
pod/velero-595b858947-z8hjl                             1/1     Running   0          26s

NAME                                                       TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)    AGE
service/openshift-adp-controller-manager-metrics-service   ClusterIP   172.30.103.211   <none>        8443/TCP   9h
service/openshift-adp-velero-metrics-svc                   ClusterIP   172.30.10.0      <none>        8085/TCP   8h

NAME                        DESIRED   CURRENT   READY   UP-TO-DATE   AVAILABLE   NODE SELECTOR   AGE
daemonset.apps/node-agent   3         3         3       3            3           <none>          27s

NAME                                               READY   UP-TO-DATE   AVAILABLE   AGE
deployment.apps/openshift-adp-controller-manager   1/1     1            1           9h
deployment.apps/velero                             1/1     1            1           8h

NAME                                                          DESIRED   CURRENT   READY   AGE
replicaset.apps/openshift-adp-controller-manager-58558fcf5f   1         1         1       9h
replicaset.apps/velero-589c78c5ff                             0         0         0       7h18m
replicaset.apps/velero-595b858947                             1         1         1       28s
replicaset.apps/velero-648dc6488d                             0         0         0       6h13m
replicaset.apps/velero-6bd8d5cb5d                             0         0         0       7h5m
replicaset.apps/velero-b868688d9                              0         0         0       6h3m
replicaset.apps/velero-bb8c78f94                              0         0         0       8h
replicaset.apps/velero-d6778cb64                              0         0         0       7h50m
`


This above Output should be used for 1.3 DPA examples.